LICENSES/MIT.txt

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+<year> <copyright holders>
```

Comment:
Not sure if it's left blank on purpose
```suggestion
<2024> <Nextcloud GmbH and Nextcloud contributors>
```

Reply:
Yes, left blank on purpose 👍

Reply:
I think it's blank on purpose. The authors of the corresponding SPDX headers are supposed to be substituted.